src/main/resources/skyblocker.mixins.json

```diff
@@ -50,5 +50,8 @@
   ],
   "injectors": {
     "defaultRequire": 1
-  }
+  },
+  "mixins": [
+    "accessor.SlotAccessor"
+  ]
```

Comment:
You probably generated this accessor, which is fine, but just remember to put it in `client` with the other mixins